src/app/form/(formSteps)/components/LoadingSpinner.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+import loadingIcon from "public/loading.gif";
+
+export const LoadingSpinner = ({
+  loading,
+  children,
+}: {
+  loading: boolean;
+  children: React.ReactNode;
+}) => {
+  return loading ? (
+    <div style={{ textAlign: "center" }} aria-live="polite">
```

Comment:
```suggestion
    <div style={{ textAlign: "center" }} aria-live="polite" aria-atomic="true" aria-busy="true | false">
```

Add aria-busy state for the loading states